psignifit/_priors.py

```diff
@@ -170,8 +170,6 @@ def setup_priors(custom_priors, bounds, stimulus_range, width_min, width_alpha,
         priors.update(custom_priors)
     check_priors(priors, stimulus_range, width_min)
 
-    for parameter, prior in priors.items():
-        priors[parameter] = normalize_prior(prior, bounds[parameter])
+    for parameter, bound in bounds.items():
+        priors[parameter] = normalize_prior(priors[parameter], bound)
     return priors
```

Comment:
I am not sure I understand why this was changed...

Reply:
I assume this is a relict from a time when bounds might not be defined for fixed items. Therefore, the former version would crash. If I'm right, you changed the behavior last week such that fixed parameters fill bounds?!